package.json

```diff
@@ -89,5 +88,8 @@
         false
       ]
     ]
+  },
+  "dependencies": {
+    "@electron/notarize": "^1.2.3"
```

Comment:
Does this need to be a production dependency?

Reply:
Nope!